Rock-paper-scissors/script.js

```diff
@@ -25,16 +25,44 @@ function getComputerSelection() {
 
 function playRound(playerSelection, computerSelection) {
   if (playerSelection === computerSelection) {
-    return `Computer chose ${playerSelection} too. It's a tie!`;
+    return `<br>Computer chose ${playerSelection} too</br> <br>It's a tie!</br>`;
   } else if (
     playerSelection === "rock" && computerSelection === "scissors"
     || playerSelection === "paper" && computerSelection === "rock"
     || playerSelection === "scissors" && computerSelection === "paper"
   ) {
-    return `Computer chose ${computerSelection} \nYou win, ${playerSelection} beats ${computerSelection}`;
-  } else { 
-    return `Computer chose ${computerSelection} \nYou lose, ${computerSelection} beats ${playerSelection}`;
+    playerWins++
+    return `<br>Computer chose ${computerSelection}</br> <br>You win, ${playerSelection} beats ${computerSelection}</br>`;
+  } else {
+    computerWins++ 
+    return `<br>Computer chose ${computerSelection}</br> <br>You lose, ${computerSelection} beats ${playerSelection}</br>`;
   }
 }
 
-main()
+function updateRoundCounter() {
+  roundCounter++;
+  if (roundCounter === 5) {
+    determineWinner();
+  }
+}
+
+function determineWinner() {
+  let winner;
+  if (playerWins > computerWins) {
+    winner = "Player";
+  } else if (computerWins > playerWins) {
+    winner = "Computer";
+  } else {
+    winner = "AI alignment";
+  }
+
+  const winnerMessage = `<br>Game over! ${winner} wins after 5 rounds</br> 
+  <br>Player: ${playerWins} <div>Computer: ${computerWins}</div></br>
+  Choose rock, paper or scissors once again to start new game`;
```

Comment:
Кажется, тут не хватает отступа. Такие отступы нужны, чтобы мы понимали, что мы находимся внутри объекта, объявление которого началось где-то выше:
```suggestion
  const winnerMessage = `<br>Game over! ${winner} wins after 5 rounds</br> 
    <br>Player: ${playerWins} <div>Computer: ${computerWins}</div></br>
    Choose rock, paper or scissors once again to start new game`;
```
Однако мне не нравится тогда расположение кавычек. В питоне есть множество способов объявить строку, которая длиннее 80 символов, например:
```python3
cool_story = (
    'This is a long story that tells us about a man named Hat. Hat was a bright'
    ' young teenager who skated, drank bourbon mixed with coke and did'
    ' every other cool shit. But then "yege" happened. What\'s "yege", you wonder?'
)
```
Думаю, что в JS'е есть какой-нибудь подобный механизм, который позволяет чётко видеть, что написано в конкретной строке и внутри чего мы сейчас находимся.